acl/src/main/java/org/springframework/security/acls/jdbc/AclClassIdUtils.java

```diff
@@ -36,10 +41,18 @@
 	private ConversionService conversionService;
 
 	public AclClassIdUtils() {
+		this(null);
```

Comment:
This means now that the conversion service can never be `null`. That means that all the checks for `hasConversionService` are no longer necessary.

Would you mind creating a ticket to improve that (remove the `hasConversionService` checks) so that we remember for the future?